### PR TITLE
Add: Ethiopia Tigray Ghent-Labeled points

### PR DESCRIPTION
**Pull request by bhyeh**
Ground truth labels of Ethiopia Tigray collected by Ghent researchers. Add for spectral signal analysis between crop and non crop points.

### CODE REVIEW

**Comment by bhyeh** [comment]
Failed - forgot to include raw data files

File: data/raw.dvc [hunk 1]
```diff
@@ -1,5 +1,5 @@
 outs:
-- md5: df1762135f5161b135cc4e664242b1ea.dir
-  size: 437588778
-  nfiles: 364
+- md5: 0da22d1871320983c53fe8182e55f514.dir
+  size: 26380
+  nfiles: 1
   path: raw
```
Comment (bhyeh) [inline data/raw.dvc:5]: Oops - this should not have been pushed w/o pulling from data/raw first.

**Comment by ivanzvonkov** [comment]
Looks  like the data pipeline has been successfully kicked off inside Test / test and Earth Engine are being exported

File: data/report.txt [hunk 2]
```diff
@@ -324,3 +324,10 @@ HawaiiCorrectiveGuided2020 (Timesteps: 24)
 ----------------------------------------------------------------------------
 eo_data_complete    329
 ✔ training amount: 329, positive class: 21.0%
+
+
+
+EthiopiaTigrayGhent2021 (Timesteps: 12)
+----------------------------------------------------------------------------
+eo_data_exporting    161
+✖ validation: 161 labels, but 0 features
```
Comment (ivanzvonkov) [inline data/report.txt:333]: As time passes and these eo data becomes available the 0 features should grow to 161 features

File: data/report.txt [hunk 3]
```diff
@@ -324,3 +324,10 @@ HawaiiCorrectiveGuided2020 (Timesteps: 24)
 ----------------------------------------------------------------------------
 eo_data_complete    329
 ✔ training amount: 329, positive class: 21.0%
+
+
+
+EthiopiaTigrayGhent2021 (Timesteps: 12)
+----------------------------------------------------------------------------
+eo_data_complete    161
+✔ validation amount: 161, positive class: 78.9%
```
Comment (bhyeh) [inline data/report.txt:333]: Looks like it is complete!

File: data/raw.dvc (hunk 1 above)
Comment (ivanzvonkov) [reply data/raw.dvc:5]: Looks like it's fixed!

**Review by ivanzvonkov — APPROVED** [review]
LGTM!

**Pull request merged**